hip/static/styles/includes/sidebar.scss

```diff
@@ -0,0 +1,24 @@
+@import "../colors";
+
+.sidebar-border-scss {
```

Comment:
I was thinking we should have a way to distinguish between `bulma` or which every style-guide we are following and the custom css that we are writing. I just wanted give the developer a clue as to where they could look for a relative class.

Reply:
That is a great idea! I usually use `grep` for this, but I see how valuable it would be to have that separation clear up front. Maybe a `-hip` prefix would make more sense to my brain than `-scss`, but either is fine. Could you add a note to the README for future developers to know about this convention, and to remind us to stick to it?

Reply:
done and updated README